tests/framework/i18n/FormatterTest.php

```diff
@@ -293,18 +294,26 @@ public function lengthDataProvider()
     /**
      * @dataProvider lengthDataProvider
      */
-    public function testIntlAsLength($message, $arguments, $expected)
+    public function testIntlAsLength($message, $arguments, $expected, $_, $expectedException = [])
```

Comment:
Why variable named `$_`?

Reply:
It contains shotLength expectation and is unused in this test.

Reply:
Fact that it is unused does not justifies such variable naming.